downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,931 @@
 layout: ballerina-blank-page
 title: Release Note
 ---
-### Overview of jBallerina {{ site.data.stable-latest.metadata.version }}
```

Comment:
We can add a small intro like this before the overview - 
This is the third alpha in a series of planned alphas and betas leading up to the Ballerina Swan Lake GA release.

Reply:
Fixed